saleor/core/tests/test_metadata_manager.py

```diff
@@ -0,0 +1,131 @@
+from sympy.testing.pytest import raises
+
+from saleor.core.models import ModelWithMetadata
```

Comment:
For code consistency, you should use relative imports.

Reply:
Fixing + opening a discussion about adding this to linter (https://docs.astral.sh/ruff/rules/relative-imports/) and hopefully changing to absolutes ;)